doc/genesis_file.md

```diff
@@ -52,8 +53,9 @@ There are multiple _parts_ in the genesis file:
 
 | option | format | description |
 |:-------|:-------|:------------|
-| `block0-date` | number | the official start time of the blockchain, in seconds since UNIX EPOCH |
+| `block0_date` | number | the official start time of the blockchain, in seconds since UNIX EPOCH |
 | `discrimination` | string | `production` or `test` |
+| `discrimination` | string | `bft` |
```

Comment:
Surely this should be `block0_consensus`?

Reply:
thanks for spotting this